testsuite/features/step_definitions/command_steps.rb

```diff
@@ -180,7 +180,11 @@
 
 When(/^I query latest Salt changes on "(.*?)"$/) do |host|
   node = get_target(host)
-  result, return_code = node.run("LANG=en_US.UTF-8 rpm -q --changelog salt")
+  salt = 'salt'
+  if host != 'server' && host != 'proxy'
```

Comment:
```suggestion
  if host != 'server'
```

Reply:
I did it different as I do not know what rubocop will say about this